<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebScript</class>
 <widget class="QWidget" name="WebScript">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WebScript</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Main">
      <attribute name="title">
       <string>Main</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="urlEdit">
           <property name="text">
            <string>about:blank</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stopButton">
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="refreshButton">
           <property name="text">
            <string>Refresh</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="infoWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>UserScript</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Userscipt:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="userscriptEdit">
         <property name="plainText">
          <string>// ==UserScript==
// @name         Userscript!
// @version      0.1
// @description  This userscript adds a &quot;Hello, world!&quot; message to the top of every web page.
// @match        https://*/*
// @grant        none
// ==/UserScript==
</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="injectButton">
         <property name="text">
          <string>Inject</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Run Script</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Script</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="scriptEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="evalButton">
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="evalResultWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Page Html</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QRadioButton" name="plainButton">
         <property name="text">
          <string>Plain Html</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="htmlBroswer">
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
